src/main/java/org/craftercms/studio/impl/v1/repository/alfresco/AlfrescoContentRepository.java

```diff
@@ -895,9 +892,13 @@ protected boolean createMissingFoldersCMIS(String fullPath) {
                     Map<String, String> newFolderProps = new HashMap<String, String>();
                     newFolderProps.put(PropertyIds.OBJECT_TYPE_ID, "cmis:folder");
                     newFolderProps.put(PropertyIds.NAME, name);
-                    Folder newFolder = folder.createFolder(newFolderProps);
-                    Property property = newFolder.getProperty("alfcmis:nodeRef");
-                    newFolderRef = property.getValueAsString();
+                    try {
+                        Folder newFolder = folder.createFolder(newFolderProps);
+                        Property property = newFolder.getProperty("alfcmis:nodeRef");
+                        newFolderRef = property.getValueAsString();
+                    } catch (CmisContentAlreadyExistsException exc) {
+                        logger.info("Folder " + cleanPath + " already exists");
```

Comment:
This should not be an info.  Also, if you saw this in the logs you would have no idea what the context is, even as a warning it is not useful.  I'd remove it all together.